cmd/dexctl/driver.go

```diff
@@ -1,13 +1,12 @@
 package main
 
 import (
-	"github.com/coreos/dex/connector"
 	"github.com/coreos/go-oidc/oidc"
 )
 
 type driver interface {
```

Comment:
Do we still find this interface useful? Could we drop it and just have different paths for different backends?